servers/zts/src/main/java/com/yahoo/athenz/zts/utils/ZTSUtils.java

```diff
@@ -56,12 +58,16 @@
     public static final String ZTS_CERT_DNS_SUFFIX =
             System.getProperty(ZTSConsts.ZTS_PROP_CERT_DNS_SUFFIX, ZTSConsts.ZTS_CERT_DNS_SUFFIX);
 
-    private static final String ATHENZ_PROP_KEYSTORE_PASSWORD      = "athenz.ssl_key_store_password";
-    private static final String ATHENZ_PROP_TRUSTSTORE_PASSWORD    = "athenz.ssl_trust_store_password";
-    private static final String ATHENZ_PROP_KEYSTORE_PATH          = "athenz.ssl_key_store";
-    private static final String ATHENZ_PROP_KEYSTORE_TYPE          = "athenz.ssl_key_store_type";
-    private static final String ATHENZ_PROP_TRUSTSTORE_PATH        = "athenz.ssl_trust_store";
-    private static final String ATHENZ_PROP_TRUSTSTORE_TYPE        = "athenz.ssl_trust_store_type";
+    private static final String ATHENZ_PROP_KEYSTORE_PASSWORD                   = "athenz.ssl_key_store_password";
+    private static final String ATHENZ_PROP_TRUSTSTORE_PASSWORD                 = "athenz.ssl_trust_store_password";
+    private static final String ATHENZ_PROP_KEYSTORE_PATH                       = "athenz.ssl_key_store";
+    private static final String ATHENZ_PROP_KEYSTORE_TYPE                       = "athenz.ssl_key_store_type";
+    private static final String ATHENZ_PROP_TRUSTSTORE_PATH                     = "athenz.ssl_trust_store";
+    private static final String ATHENZ_PROP_TRUSTSTORE_TYPE                     = "athenz.ssl_trust_store_type";
+    private static final String ATHENZ_PROP_LOCAL_CERT_PATH_FOR_SSL             = "athenz.ssl_local_cert_path";
+    private static final String ATHENZ_PROP_LOCAL_KEY_PATH_FOR_SSL              = "athenz.ssl_local_key_path";
+    private static final String ATHENZ_PROP_USE_LOCAL_CERT_AND_KEY_FOR_SSL      = "athenz.ssl_use_local_cert_and_key";
```

Comment:
in the next two settings let's use the file instead of local, e.g.:

private static final String ATHENZ_PROP_USE_FILE_CERT_AND_KEY_FOR_SSL = "athenz.ssl_client_use_file_cert_and_key";

Reply:
Fixed